CP02PMCI/CP02PMCI_D00010_ingest.csv

```diff
@@ -1,7 +1,7 @@
 parser,filename_mask,reference_designator,data_source,status,notes
 mi.dataset.driver.adcps_jln.stc.adcps_jln_stc_telemetered_driver,/omc_data/whoi/OMC/CP02PMCI/D00010/imm/adcpt/adcpt_*_*.DAT,CP02PMCI-RII01-02-ADCPTG010,telemetered,Available,
-#mi.dataset.driver.cg_stc_eng.stc.cg_stc_eng_stc_telemetered_driver,/omc_data/whoi/OMC/CP02PMCI/D00010/irid2shore/stc_status.*_*.txt,CP02PMCI-SBS01-00-STCENG000,telemetered,Available,Not ingested yet due to system error
-#mi.dataset.driver.cg_stc_eng.stc.mopak_o_dcl_telemetered_driver,/omc_data/whoi/OMC/CP02PMCI/D00010/3dmgx3/*_*.3dmgx3.log,CP02PMCI-SBS01-01-MOPAK0000,telemetered,Available,Not ingested yet due to system error
+mi.dataset.driver.cg_stc_eng.stc.cg_stc_eng_stc_telemetered_driver,/omc_data/whoi/OMC/CP02PMCI/D00010/irid2shore/stc_status.*_*.txt,CP02PMCI-SBS01-00-STCENG000,telemetered,Available,
+#mi.dataset.driver.cg_stc_eng.stc.mopak_o_dcl_telemetered_driver,/omc_data/whoi/OMC/CP02PMCI/D00010/3dmgx3/*_*.3dmgx3.log,CP02PMCI-SBS01-01-MOPAK0000,telemetered,Not available,Data not available 
 mi.dataset.driver.WFP_ENG.STC_IMODEM.wfp_eng_stc_imodem_telemetered_driver,/omc_data/whoi/OMC/CP02PMCI/D00010/imm/mmp/E*.DAT,CP02PMCI-WFP01-00-WFPENG000,telemetered,Available,
 mi.dataset.driver.vel3d_k.wfp_stc.vel3d_k_wfp_stc_telemetered_driver,/omc_data/whoi/OMC/CP02PMCI/D00010/imm/mmp/A*.DEC,CP02PMCI-WFP01-01-VEL3DK000,telemetered,Available,
 mi.dataset.driver.dofst_k.wfp.coastal_dofst_k_wfp_telemetered_driver,/omc_data/whoi/OMC/CP02PMCI/D00010/imm/mmp/C*.DAT,CP02PMCI-WFP01-02-DOFSTK000,telemetered,Available,

```

Comment:
Uncommented stc status data line because those data are available and were ingested.
Confirmed the mopak data were not available so I changed the status and added a comment